### PR TITLE
docs: Add example for passing data from server to component

### PR DESCRIPTION
**Pull request by saviomuc**
This pull request closes issue #802 

- [x] All commit messages adhere to the [appropriate format](https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#commit) in order to trigger a correct release
- [ ] All code is written in untranspiled ECMAScript (ES 2017) and is formatted using [prettier](https://prettier.io)
- [ ] Necessary unit tests are added in order to ensure correct behavior
- [x] Documentation has been added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1117,6 +1117,39 @@ class MyExpressMixin extends Mixin {
...
+```javascript
+enhanceServerData(data, req, res) {
+  return { ...data, someKey: res.locals.someKey };
+}
+```
+
+**`my-component.js`**
+
+```javascript
+const MyComponent = ({serverData}) => <div>{serverData.isMobile}</div>
```
Comment (ZauberNerd) [inline README.md]: Shouldn't this be `serverData.someKey` as seen in the example above?

Comment (saviomuc) [reply README.md]: Valid! I changed it accordingly.

**Comment by jhiode** [comment]
Hey, thanks for the PR!

In my opinion we should also document the drawbacks of injected data. I have at least two in mind:
1. the response might not be cacheable anymore because the data is request-specific.
2. the injected data can get stale easily because a SPA does not get reloaded as often as a classical page.

**Comment by saviomuc** [comment]
Hi @jhiode I am not sure if that's the correct scope for this.

The purpose of this PR is just to clarify the workflow to pass data from server. 
While the issues you list might be related imho they also introduce noise which is not directly related to the documentation problem I like to solve. 

I'd rather change this PR to a more simple example instead of introducing a discussion of potential problems.

**Pull request merged**